lib/js/components/RichList/BasicRichListItem/BasicRichListItem.vue

```diff
@@ -41,8 +41,8 @@
 				</div>
 			</div>
 		</template>
-		<template v-if="$slots.meta" #meta>
-			<slot name="meta" />
+		<template v-if="$slots.metadata" #metadata>
+			<slot name="metadata" />
```

Comment:
Remember to release as a major version and fix https://github.com/bethinkpl/wnl-platform/blob/a87a7b11087bbfc7c3ae4656d1a1a7b693c417bc/resources/assets/js/admin/modules/mediaManager/components/MediaGallery/MediaSelectedAsset.vue#L49

Reply:
👍 
accept this PR and I'll proceed with required changes ;)